### PR TITLE
Remove sudo

### PR DESCRIPTION
**Pull request by lephino**
This should be merged AFTER the pre/post plays.

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -17,6 +56,32 @@
         - splunk.role != 'splunk_search_head_captain'
         - not splunk.upgrade
 
+    # the get_url module is for linux only
+    - name: Check if Post plays need downloaded
+      when:
+        - ansible_post_tasks is defined
+        - ansible_post_tasks is not none
+        - ansible_system is match("Linux")
+      get_url:
```
Comment (nwang92) [inline site.yml]: Do we want retries over these pre-/post- plays?

**Comment by jmervine** [comment]
Getting the following error when using this branch
```
cat: /opt/ansible/inventory/ansible_inventory.json: No such file or directory
```

**Comment by lephino** [comment]
> Getting the following error when using this branch
> 
> ```
> cat: /opt/ansible/inventory/ansible_inventory.json: No such file or directory
> ```

Were you running this through the container @jmervine, or did you get that error with just the splunk-ansible plays?  Did you also use the sudo branch on the container if it was in the container?

**Comment by jmervine** [comment]
@lephino Through the container. I built splunk/docker-splunk and "remove_sudo" as the ansible branch.

**Comment by jmervine** [comment]
@lephino Are you on splunk-usergroups.slack.com?

**Comment by jmervine** [comment]
This might be an issue with splunk/docker-splunk master, because now I'm getting that error with develop ansible as well.

**Comment by jmervine** [comment]
Hmmm... doesn't seem to be any recent merges though.

**Comment by jmervine** [comment]
Just noticed https://github.com/splunk/docker-splunk/pull/72/. Let me try building them together.

**Comment by lephino** [comment]
> Hmmm... doesn't seem to be any recent merges though.

It was something i noticed too on master, in the sudo play you'll see I added the redirect for 2> so that we don't see them.  I'll make sure it's gone before 7.2.2.  I think 

> @lephino Are you on splunk-usergroups.slack.com?

"bbingham" over there, feel free to ping me

File: site.yml (hunk 1 above)
Comment (lephino) [reply site.yml]: Didn't want to delay the startup if they aren't url's.  I'm torn on it, lets wait and see if anyone asks for it.

**Pull request merged**